uk_bin_collection/uk_bin_collection/councils/WyreForestDistrictCouncil.py

```
@@ -1,62 +1,138 @@
+
 from uk_bin_collection.uk_bin_collection.common import *
 from uk_bin_collection.uk_bin_collection.get_bin_data import AbstractGetBinDataClass
 
+QUERY_URL = "http://www.wyreforest.gov.uk/querybin.asp"
+
+DAYS_OF_WEEK = [
+    "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"
+]
+
+
+def _next_weekday(day_name: str, from_date: datetime = None) -> datetime:
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)